components/layout/RegisterForm.js

```diff
@@ -0,0 +1,109 @@
+import { useState } from "react";
+import { useRouter } from "next/router";
+import styled from "styled-components";
+import Button from "../ui/Button";
+import Input from "../ui/Input";
+import { showToast } from "../ui/ToastMessage";
+
+const Container = styled.div`
+  min-height: 100vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`;
+
+const FormCard = styled.div`
+  background: white;
+  padding: 2rem;
+  border-radius: 8px;
+  box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1);
+  width: 400px;
+`;
+
+const Title = styled.h1`
+  font-size: 1.5rem;
+  font-weight: bold;
+  margin-bottom: 1.5rem;
+`;
+
+const FormGroup = styled.div`
+  margin-bottom: 1rem;
+`;
+
+function RegisterForm() {
+  const router = useRouter();
+  const [userData, setUserData] = useState({
+    name: "",
+    email: "",
+    password: "",
+  });
+
+  const handleSubmit = async (e) => {
```

Comment:
please check your code consistency across your entire PR, regarding abbreviations and syntax.